pages/index.js

```diff
@@ -24,7 +24,7 @@ const modalStates = Object.freeze({
 
 // Page: Events
 export default function Events({ session }) {
-  const { setEvent } = Navigation.useContainer();
+  const [, setEvent] = Event.useContainer();
```

Comment:
Why ,?

Reply:
We were using `Navigation` just to store the id of the currently selected event. For the feedback sharing, I'd assume that we would have an extra field in the Event object storing any fields related to feedback status, which Feedback Sharing depends on. Also, naming it Navigation was not so clear as to what it was for.